### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by andrunko**
Fixes for some small issues found while testing.

### CODE REVIEW

File: app/flatpak-cli-transaction.c [hunk 1]
```diff
@@ -248,7 +248,7 @@ progress_changed_cb (FlatpakTransactionProgress *progress,
 {
   FlatpakCliTransaction *cli = data;
   FlatpakTransaction *self = FLATPAK_TRANSACTION (cli);
-  FlatpakTransactionOperation *op = flatpak_transaction_get_current_operation (self);
+  g_autoptr(FlatpakTransactionOperation) op = flatpak_transaction_get_current_operation (self);
```
Comment (uajain) [inline app/flatpak-cli-transaction.c]: I didn't find `G_DEFINE_AUTOPTR_CLEANUP_FUNC` defined for `FlatpakTransactionOperation` in the codebase.
You need to define:
`G_DEFINE_AUTOPTR_CLEANUP_FUNC (FlatpakTransactionOperation, g_object_unref);` in flatpak-transaction.h header file I think.

Comment (wjt) [reply app/flatpak-cli-transaction.c]:
```
common/flatpak-transaction.h:89:G_DECLARE_FINAL_TYPE (FlatpakTransactionOperation, flatpak_transaction_operation, FLATPAK, TRANSACTION_OPERATION, GObject)
```

[`G_DECLARE_FINAL_TYPE`](https://developer.gnome.org/gobject/stable/gobject-Type-Information.html#G-DECLARE-FINAL-TYPE:CAPS):

> This results in the following things happening:
> […]
> * g_autoptr() support being added for your type, based on the type of your parent class

**Comment by pwithnall** [comment]
bot, test pull request

**Comment by pwithnall** [comment]
drat, looks like I don’t have the right privileges to boss the bot around

**Comment by matthiasclasen** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**